ui/apps/platform/src/Containers/ComplianceEnhanced/ClusterCompliance/ScanConfigs/Wizard/ProfileSelection.tsx

```diff
@@ -49,61 +69,89 @@ function ProfileSelection({ profiles, isFetchingProfiles }: ProfileSelectionProp
             .filter((_, index) => {
                 return index === rowId ? isSelected : selected[index];
             })
-            .map((profile) => profile.id);
+            .map((profile) => profile.name);
 
         setFieldValue('profiles', newSelectedIds);
     };
 
     const handleSelectAll = (event: React.FormEvent<HTMLInputElement>, isSelected: boolean) => {
         onSelectAll(event, isSelected);
 
-        const newSelectedIds = isSelected ? profiles.map((profile) => profile.id) : [];
+        const newSelectedIds = isSelected ? profiles.map((profile) => profile.name) : [];
```

Comment:
What are pro and con to rename variable? There are only 4 occurrences: 2 here and 2 in function above.

Reply:
I'm a little confused by your question. I had to change from id to name because id isn't included in the response. Before I was using mock data based off a different proto but we've since added the "summary" endpoint where name is unique

Reply:
Aha, I meant `newSelectedIds` versus `newSelectedNames` as variable name.

Reply:
Oh good idea